docs/custom-root-ca.md

```diff
@@ -31,3 +43,16 @@ docker run --rm -it \
   -e NODE_EXTRA_CA_CERTS=/my-root-ca.crt \
   containerbase/buildpack bash
 ```
+
+### Java
+
+For Java you need to mount your own certificate store to `/opt/buildpack/ssl/cacerts`.
```

Comment:
How is this URL used when it's only mapped at runtime? e.g. will any java we install automatically look for this URL even if it wasn't present at the time of prepare-tool?

Reply:
- `/opt/buildpack/ssl` is created by `install-buildpack`
- `/opt/buildpack/ssl/cacerts` is created by `prepare-tool java` if not exists
- when running `install-tool java <version>` as root, it automatically runs `prepare-tool java`

Reply:
So the certs file always exists and is expected by Java to be present? That way when it's overridden at runtime Java is already configure to use it?

Reply:
yes, it's symlinked to the normal path
https://github.com/containerbase/buildpack/blob/5aed1e46fb27b0b03a1aaa4fd583546d24e11e4c/src/usr/local/buildpack/utils/java.sh#L74-L78